src/getting-started/resources.md

```diff
@@ -4,35 +4,17 @@ title: Resources
 
 Magento provides a wealth of business and technical resources, self-help tools, and services to help you succeed.
 
-[Product Documentation][1]{:target="_blank"}
-:  Whether you’re a merchant, designer, [developer][2], or all of the above, this is where you’ll gain a deeper insight into how you can leverage the power of Magento to grow your business.
-
-[Magento Resources Library][3]{:target="_blank"}
-:  Gain access to expert insight and online business resources to help develop and improve your store. Ebooks, Magento whitepapers, and webinars.
-
-[Security Center][4]{:target="_blank"}
-:  Join Magento’s Security Alert Registry to get the latest information on potential vulnerabilities and [Security Best Practices][5].
-
-[Forums][6]{:target="_blank"}
-:  The Magento forums provide access to a network of dedicated Magento enthusiasts who share tips, tricks and support.
-
-[Blog][7]{:target="_blank"}
-:  Check out the Magento blog for the latest information about new features, trends, best practices, upcoming events, and more!
-
-[Events and Webinars][8]{:target="_blank"}
-:  Keep tabs on our calendar of upcoming events and webinars. There's always something happening in the Magento ecosystem!
-
-[Magento Marketplace][9]{:target="_blank"}
-:  Visit Magento’s next-generation marketplace of digital commerce solutions. Magento Marketplace offers merchants a curated selection of solutions, while providing qualified developers the tools, platform, and prime location for a thriving business.
-
-[Training &amp; Certification][10]{:target="_blank"}
-:  Learn how to unleash the power and flexibility of Magento. We provide training for every role in your business, including marketers, designers, developers, and more. Experienced Magento professionals can validate their real-world skills by earning Magento certification. [][11]
-
-[Partners][12]{:target="_blank"}
-:  Magento partners are committed to your success, and provide custom integrations, best-in-class customer experiences, strategic marketing initiatives, and expert performance and scalability optimization for both on-premise and cloud-based solutions.
-
-[Expert Consulting Services][13]{:target="_blank"}
-:  Our Expert Consulting Group (ECG) helps Magento merchants and Solution Partners maximize their success. Our experts offer comprehensive analysis and best practice recommendations, from architecture planning through post-deployment.
+[Product documentation][1]{:target="_blank"}|Whether you are a merchant, designer, [developer][2], or all of the above, this is where you can gain a deeper insight into how you can leverage the power of Magento to grow your business.|
+[Magento Resources Library][3]{:target="_blank"}|Gain access to expert insight and online business resources to help develop and improve your store. Ebooks, Magento whitepapers, and webinars.|
```

Comment:
is store, ebooks, whitepapers all part of the same list?  if so, change there . after store.